### DISCUSSION TITLE
MetaSpore on AlphaIDE使用体验

### DISCUSSION THREAD

**Discussion by Jingrui92** [meta-soul/MetaSpore#128] [General]
您所处的组织、公司或学校: 小外企
您所在的地区：北京
您使用/期待使用数元灵产品的场景：搜推广
您感兴趣/期待的功能点：一站式的离线开发、线上部署
您的建议或反馈：
  1. 平台易用性：metaSpore提供了在线IDE等多种开发方式，可以兼容使用者不同的开发习惯，这一点非常贴心；
  2. 开发使用角度：尝试了下离线机器学习的开发训练，从使用过程来看，平台提供的功能非常丰富，对于从零启动一个机器学习项目非常的友好，几乎涵盖了支撑所有需求的功能。在线Notebook也相当稳定，不会出现断线的情况。且由于很好集成了Git，代码托管也非常方便。在此基础上，数据EDA分析到开发调试模型上线都非常的友好。
  3. 入门落地：目前来看，从用户注册到跑通第一个项目的过程中，平台还存在着一点小小的问题。比如缺乏细致的指引文档，且还会存在一些小的bug，比如申请资源fail，存储权限fail，安装第三方环境fail等。可能这块缺乏更容易得到的用户指南。
  4. UI设计: 从个人角度来说，感觉目前平台的注册网页和IDE设计还较为朴素，在注册账户时，邮箱输入栏里甚至还会疑似将部分字母显示为“全角”字符，这一点在使用时会对平台的精致度有些伤害。
  5. 总结：我觉得metaSpore平台是一个非常优秀的平台，取大家所长，在各个方面都存在着显著的设计上的优势，但现阶段有些细节可能还需要打磨一下，总之是属于会继续尝试使用的平台~
  
![image](https://user-images.githubusercontent.com/102157531/172051887-4623bb54-74e7-4c59-8ed1-ed4a462c137c.png)
![image](https://user-images.githubusercontent.com/102157531/172051918-9cac49c8-96b2-4ee3-bbce-50f392cfd764.png)
![image](https://user-images.githubusercontent.com/102157531/172052086-496448ce-0b2f-4abc-abcf-a4d112b1fd61.png)

**Comment by intelligencegear** [comment]
感谢您的反馈，从截图上看：
1. **资源问题**：可能目前测试集群中的资源池并不充分，这个会和运维开发同学通知一下；
2. **权限问题**：应该是在新建Notebook的时候，AWS Conf选项没勾选，所以读写S3数据时有报错，这个之后会考虑在下一个迭代中默认勾选这个配置，避免大家测试过程中遇到相似问题；
3. **交互问题**：感谢您的反馈，后面我们的会着力改善软件的易用性。

若上述所说，您所提到关于在AlphaIDE中操作时，遇到的资源、权限、交互的问题反馈我们已经收到，相信会在下一个迭代中得到改善。
